src/App.jsx

```diff
@@ -45,6 +46,17 @@ const router = createBrowserRouter(
         </Layout>
       ),
     },
+    {
+      path: '/exhibitions/:exhibitionId',
+      element: (
+        <Layout>
+          <div>
```

Comment:
I don't think this `div` is necessary. If the UI looks still good after getting rid of it, please go ahead and also delete the one from the route with path `/art-works/:artWorkId`.